libs/designsystem/src/lib/components/popover/popover.component.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../scss/utils';
+
+:host {
+  display: none;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: z('popover');
+}
+
+:host(.is-opening) {
+  display: block;
+  visibility: hidden;
+}
+
+:host(.is-open) {
+  display: block;
+}
+
+.wrapper {
+  position: fixed;
+  margin-top: size('xxxs');
```

Comment:
Doesn't this add too much top margin? (and bottom) 

It has more margin than the ones without popover. 

Probably worth talking to UX about.

Reply:
The answer I got was that the distance between button & popover Is 8 px. Therefore I will just leave as is, as this is the new distance when setting margin-top.